testing/functional/tests/cypress/page-objects/Institution-objects/InstitutionProfileObject.ts

```diff
@@ -8,11 +8,11 @@ export default class InstitutionProfileObject {
   }
 
   legalOperatingNameInputText() {
-    return cy.get("#ek5v67e-institutionLegalName");
+    return cy.get("[name='data[institutionLegalName]']");
```

Comment:
We should have the `data-cy` in place for these items now.

Reply:
Yes, however this selector (data-cy) is still not changed in the live test or dev URL. I'll update selector once its done in live test or dev URL.